engine/flink/executor/src/main/scala/pl/touk/nussknacker/engine/process/ProcessNameValidator.scala

```diff
@@ -0,0 +1,31 @@
+package pl.touk.nussknacker.engine.process
+
+import com.typesafe.config.Config
+import pl.touk.nussknacker.engine.{CustomProcessValidator, CustomProcessValidatorFactory}
+import pl.touk.nussknacker.engine.api.context.ProcessCompilationError
+import pl.touk.nussknacker.engine.api.context.ProcessCompilationError.ProcessNameValidationError
+import pl.touk.nussknacker.engine.canonicalgraph.CanonicalProcess
+
+import scala.concurrent.Future
+
+class ProcessNameValidatorFactory extends CustomProcessValidatorFactory {
+  override def validator(config: Config): CustomProcessValidator = new ProcessNameValidator(config)
+}
+
+class ProcessNameValidator(config: Config) extends CustomProcessValidator {
+
+  private lazy val flinkProcessNameValidationPattern = "[a-zA-Z0-9-_ ]+"r
```

Comment:
there is no limit on length? maybe you can add some link to docs or flink class that validates it?